app/experimenter/legacy-ui/rapid/components/experiments/ResultsTable.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+
+import ResultsRow from "experimenter-rapid/components/experiments/ResultsRow";
+import { RESULT_COLUMN_TIPS } from "experimenter-rapid/components/experiments/tooltips";
+import { featureOptions } from "experimenter-rapid/components/forms/ExperimentFormOptions";
+import { AnalysisPoint, ExperimentData } from "experimenter-types/experiment";
+
+enum Significance {
+  POSITIVE,
+  NEGATIVE,
+  NEUTRAL,
+}
+
+const Metric = {
+  RETENTION: "retained",
+  SEARCH: "search_count",
+  USER_COUNT: "identity",
+};
```

Comment:
You could rename `tooltips.tsx` to `constants.ts` and move all these in there?

Reply:
I made two separate files. I think it's nice to have tooltips separate because there will be a lot of them.